### PR TITLE
fix: Rebuild instruction when "setMask" is called

### PR DESCRIPTION
**Pull request by katmai7**
##### Description of change
If mask `inverse` option changed dynamically, it has no effect.
So we need to mark render group structure as changed in order to trigger instructions rebuild.

##### Pre-Merge Checklist

- [ ] Tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [x] Lint process passed (`npm run lint`)
- [x] Tests passed (`npm run test`)

### CODE REVIEW

File: src/scene/container/container-mixins/effectsMixin.ts [hunk 1]
```diff
@@ -92,10 +102,7 @@ export const effectsMixin: Partial<Container> = {
 
         this.effects.splice(index, 1);
 
-        if (this.parentRenderGroup)
-        {
-            this.parentRenderGroup.structureDidChange = true;
```
Comment (katmai7) [inline src/scene/container/container-mixins/effectsMixin.ts:97]: I assume that when effect is removed it should has same logic as `addEffect` for getting rendering group.
Please correct me if I'm wrong.

**Comment by katmai7** [comment]
Hi @GoodBoyDigital,
It's a fix for inverse mask, could yo please review it?

File: src/scene/container/container-mixins/effectsMixin.ts [hunk 2]
```diff
@@ -48,6 +54,15 @@ export const effectsMixin: Partial<Container> = {
      */
     effects: [],
 
+    _markStructureAsChanged()
```
Comment (bigtimebuddy) [inline src/scene/container/container-mixins/effectsMixin.ts:57]: Could you add private to this?

```suggestion
    private _markStructureAsChanged()
```

File: src/scene/container/container-mixins/effectsMixin.ts [hunk 3]
```diff
@@ -21,6 +21,11 @@ export interface MaskOptions
     inverse: boolean;
 }
 
+export type MaskOptionsAndMask = MaskOptions &
+{
+    mask: Mask;
+};
```
Comment (bigtimebuddy) [inline src/scene/container/container-mixins/effectsMixin.ts:27]: Why not interface?

```suggestion
export interface MaskOptionsAndMask extends MaskOptions
{
    mask: Mask;
}
```

**Review by GoodBoyDigital — APPROVED** [review]
👍

**Pull request merged**